karma.conf-sauce.js

```diff
@@ -2,37 +2,31 @@ var _ = require('underscore');
 
 // Browsers to run on Sauce Labs platforms
 var sauceBrowsers = _.reduce([
-  ['firefox', '35'],
-  ['firefox', '30'],
-  ['firefox', '21'],
-  ['firefox', '11'],
-  ['firefox', '4'],
+  ['firefox', 'latest'],
```

Comment:
firefox have ESR (extended support release), i guess it is good idea to add it here aswell

Reply:
I wanted to do this but then I found out that Firefox ESR isn't explicitly listed on the page with supported OSs and browsers. https://saucelabs.com/platform/supported-browsers-devices

Reply:
oh, okay then